seqr/views/utils/pedigree_image_utils.py

```diff
@@ -139,15 +140,15 @@ def _update_pedigree_image(family, user, project_guid=None):
 
     if not os.path.isfile(png_file_path):
         logger.error('Failed to generate pedigree image for family {}: {}'.format(family_id, completed_process.stdout),
-                     extra={'detail': {'ped_file': ped_file_rows}})
+                     extra={'detail': {'ped_file': ped_file_rows}, 'user': user})
```

Comment:
A warning level logging should be enough.

Reply:
So unfortunately I can't figure out a way to programmatically differentiate between "failed to generate because the family structure is too complicated or sparse to successfully work" and "failed to generate because this somewhat hacky perl script is broken" so I need it to be an error and alert every time it fails so I can manually check to make sure that pedigree generation isn't TOTALLY broken. We have an open ticket to stop using this perl script for pedigree generation, and once thats done we will be able to take this out entirely